deploy/helm/kubecf/templates/_capi.tpl

```diff
@@ -0,0 +1,31 @@
+{{- /*
+==========================================================================================
+| _capi.setProperty $property $value
++-----------------------------------------------------------------------------------------
+| There are 4 CAPI jobs that all share the same cloud_controller_ng specs file,
+| and therefore potentially all use the same "cc" properties. This template will
+| set a property in all of the jobs, so there is a single location to keep track
+| of instance groups these jobs run in.
+|
+| $property can use dotted path notation to specify nested properties,
+| e.g. "diego.foo" to set the "cc.diego.foo" property.
+==========================================================================================
+*/}}
+{{- define "_capi.setProperty" }}
+  {{- $property := index . 0 }}
+  {{- $value := index . 1 }}
+
+  {{- $ig := dict }}
+  {{- $_ := set $ig "cloud_controller_ng"     "api"       }}
+  {{- $_ := set $ig "cloud_controller_worker" "cc-worker" }}
+  {{- $_ := set $ig "cloud_controller_clock"  "scheduler" }}
+  {{- $_ := set $ig "cc_deployment_updater"   "scheduler" }}
+  {{- /* XXX cc_route_syncer is not in cf-deployment; see CF-K8s-Networking */}}
+  {{- /* $_ := set $ig "cc_route_syncer" "???" */}}
+
+  {{- range $job := keys $ig }}
+- type: replace
+  path: /instance_groups/name={{ get $ig $job }}/jobs/name={{ $job }}?/properties/cc/{{ $property | replace "." "/" }}
```

Comment:
Change in a29ccb5

I don't understand why you don't think this is a fit for a `dict`. It's a mapping from job name to instance group name, which seems like a classic use case for a hash/map/dict... Since we iterate over it just once, and otherwise don't take advantage of the lookup mechanism, it is maybe overkill for the current situation, but still feels like the natural choice of data structure in general to me.

Reply:
I didn't think it was a good fit for a dict because neither instance group names nor job names are unique:
- one instance group may have multiple jobs (e.g. the `api` instance group has lots of jobs)
- one job may appear in multiple instance groups (though I have no good examples off hand; perhaps we have no overlap at the moment?)

I'm fine with leaving it as a dict at this point, since it presents no issues, but believe we may need to change it in the future.

Reply:
There is no requirement for dict values to be unique, so I didn't think that instance groups have multiple jobs could be a problem.

But you are right that potentially a job can appear in multiple instance groups as well (e.g. `cfdot` does), and I didn't think about that. I don't think it applies to any of the CAPI jobs, so we can leave this as-is for now, and change it when it becomes a problem. Thanks for pointing out my erroneous assumption!